.gitignore

```diff
@@ -25,3 +25,6 @@ build
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
+yarn-lock.json
```

Comment:
This changes are not a part of this PR, it is better to have PRs which do only do one job